html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email-textbox">
+          <span>Email</span>
+        </label>
+        <br />
+        <input type="email" name="email" size="20" maxlength="50" id="email-textbox" />
+      </div>
+      <div>
+        <span>Why is coding cool?</span>
+      </div>
+      <div>
+        <select name="reasons">
+          <option value="option">Select an Option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Becuase it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <span>Favorite Language (Choose One):</span>
+        <br />
+          <input type="radio" name="favorite-language" value="html" id="html-favorite-language"/>
+          <label for="html-favorite-language">HTML</label>
+          <input type="radio" name="favorite-language" value="css" id="css-favorite-language"/>
+          <label for="css-favorite-language">CSS</label>
+          <input type="radio" name="favorite-language" value="javascript" id="js-favorite-language"/>
+          <label for="js-favorite-language">JavaScript</label>
+      </div>
+      <div>
+        <span>Would you like to sign up for more information?</span>
+        <br />
+        <input type="checkbox" name="Sign me up!" value="Sign me up!" id="Sign Up"/>
+        <label for="Sign Up">Sign me up!</label>
```

Comment:
Make sure you don't put spaces in `for` or `id` attributes, a space means multiple values in attributes. You should use a `-` between the words. Also, keep the names all lowercase